style.css

```diff
@@ -0,0 +1,101 @@
+:root {
+  --main-radius: 5px;
+  --main-padding: 5px;
+}
+
+.container {
+  display: grid;
+  height: 100vh;
+
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 0.2fr 1.5fr 1.2fr 0.8fr;
+
+  grid-template-areas:
+    "nav nav nav nav"
+    "sidebar main main main"
+    "sidebar content1 content2 content3"
+    "sidebar footer footer footer";
+
+  grid-gap: 0.2rem;
+
+  font-family: "inter", sans-serif;
+  font-weight: 800;
+  text-transform: uppercase;
+  font-size: 0.75em;
+
+  text-align: center;
+
```

Comment:
add this in your css:
```js
padding: var(--main-padding);
```
so this will remove the scroll bar which come on your screen

Reply:
padding: var(--main-padding); added